src/istio/control/tcp/request_handler_impl.cc

```diff
@@ -55,16 +55,17 @@ CancelFunc RequestHandlerImpl::Check(CheckData* check_data,
 
 // Make remote report call.
 void RequestHandlerImpl::Report(ReportData* report_data) {
```

Comment:
I think we can remove this function. It was there when mixer_client library were at different repo.

Reply:
Sure, removed.